### PR TITLE
Using domain for drone badge

### PR DESCRIPTION
**Pull request by Hackwar**
The URL for the badge in the README.md for the drone CI has been changed to use the ci.joomla.org domain instead of the IP address.

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 279a17c8f14290fb1fb925dad8b3559e47e39a80<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25240">issues.joomla.org/tracker/joomla-cms/25240</a>.</sub>

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Build Status
 ---------------------
 | Travis-CI  | Drone-CI | AppVeyor |
 | ------------- | ------------- | ------------- |
-| [![Build Status](https://travis-ci.org/joomla/joomla-cms.svg?branch=staging)](https://travis-ci.org/joomla/joomla-cms)  | [![Build Status](http://213.160.72.75/api/badges/joomla/joomla-cms/status.svg)](http://213.160.72.75/joomla/joomla-cms)  | [![Build status](https://ci.appveyor.com/api/projects/status/bpcxulw6nnxlv8kb/branch/staging?svg=true)](https://ci.appveyor.com/project/joomla/joomla-cms)  |
+| [![Build Status](https://travis-ci.org/joomla/joomla-cms.svg?branch=staging)](https://travis-ci.org/joomla/joomla-cms)  | [![Build Status](http://ci.joomla.org/api/badges/joomla/joomla-cms/status.svg)](http://ci.joomla.org/joomla/joomla-cms)  | [![Build status](https://ci.appveyor.com/api/projects/status/bpcxulw6nnxlv8kb/branch/staging?svg=true)](https://ci.appveyor.com/project/joomla/joomla-cms)  |
```
Comment (Quy) [inline README.md]:
```suggestion
| [![Build Status](https://travis-ci.org/joomla/joomla-cms.svg?branch=staging)](https://travis-ci.org/joomla/joomla-cms)  | [![Build Status](https://ci.joomla.org/api/badges/joomla/joomla-cms/status.svg)](https://ci.joomla.org/joomla/joomla-cms)  | [![Build status](https://ci.appveyor.com/api/projects/status/bpcxulw6nnxlv8kb/branch/staging?svg=true)](https://ci.appveyor.com/project/joomla/joomla-cms)  |
```

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 8f061bcba4ce3c5bbff3894a778e933c80bfc321<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25240">issues.joomla.org/tracker/joomla-cms/25240</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8f061bcba4ce3c5bbff3894a778e933c80bfc321<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25240">issues.joomla.org/tracker/joomla-cms/25240</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25240">issues.joomla.org/tracker/joomla-cms/25240</a>.</sub>

**Comment by HLeithner** [comment]
thx

**Pull request merged**